cmd/nerdctl/system_prune.go

```diff
@@ -99,5 +100,8 @@ func systemPruneAction(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-	return imagePrune(cmd, client, ctx)
+	if err := imagePrune(cmd, client, ctx); err != nil {
+		return err
+	}
+	return builderPruneAction(cmd, args)
```

Comment:
NOLGTM

```
sudo nerdctl system prune --all
WARNING! This will remove:
  - all stopped containers
  - all networks not used by at least one container
  - all images without at least one container associated to them
  - all build cache

Are you sure you want to continue? [y/N] y
Deleted Images:
Untagged: docker.io/library/nginx:test
deleted: sha256:af32df749f3fb0040acdb0015af7b7da8f6d600572b22e9137250d3dfbe111f2
deleted: sha256:97b6c514961c6bf7c4e9bf4c14bfdcc36b507b88aad22483d28b2f9d21b2c901
deleted: sha256:1c6d36bc7d70f055c5da944961712237413705cc3674f9fb44a960b29552e8a8
deleted: sha256:8b1cace7b5b65b51ffaccdf08ff3a3628748d083ede7d314dd22cad8f7839c4c
deleted: sha256:31a3b8d1409240f584c30db43d20e0bdba94d36e0316da0ad0da7d7d91159519
deleted: sha256:4e024e3fcf12199b325c226eecb7b8bc5d5cb04ffccd16112bf9e51ea458fc44
deleted: sha256:5f70bf18a086007016e948b04aed3b82103a36bea41755b6cddfaf10ace3c6ef

ID                                                                      RECLAIMABLE     SIZE    LAST ACCESSED
5ng2p4kgza3jsz2o35vdm5xdm                                               true            8.22kB
qpaheqehsy3qit4julsunsdn7*                                              true            4.10kB
zr5ioimrfo9faxb82auoeijir*                                              true            8.19kB
w61ewoia9iv5p5pbm1fpoabbs                                               true            17.78kB
3wp3a2ozz29fqq6slzd6ljw6e                                               true            12.96kB
svf78j2ocb1lfvnjf0ai6lr53                                               true            13.18kB
8z7keywza1z3vxu7w254v6cbr                                               true            8.79kB
abq5892g9c8xjv6xvv3usv59w                                               true            88.87MB
j08w18u4sm5o6s71gdlgl32db                                               true            115.81MB
Total:  204.75MB
```

The output of `builder prune` should only contain the ID instead of the table.

Reply:
Also, `Total` can confuse users, it should be all reclaimed space.

Reply:
What should we do?
Should we extend `buildctl prune` to support JSON output? Or should we use the Go API of BuildKit?
We could use `nerdctl/pkg/tabutil.TabReader` too, but not robust.

Reply:
Go API and `TabReader` are both ok, `TabReader` is enough for now.

Reply:
Yes, TabReader is enough for now, but we should switch away to use JSON when this gets merged (BuildKit 0.11)
- https://github.com/moby/buildkit/pull/2992

Reply:
What should I do now? Use `TabReader`? or wait https://github.com/moby/buildkit/pull/2992 get merged then use JSON output

Reply:
You can use TabReader for now, and add a `TODO` comment to use https://github.com/moby/buildkit/pull/2992 in the future